docs/source/saving-and-reading-results.mdx

```diff
@@ -3,30 +3,30 @@
 ## Saving results locally
 
 Lighteval will automatically save results and evaluation details in the
-directory set with the `--output_dir` argument. The results will be saved in
+directory set with the `--output-dir` option. The results will be saved in
 `{output_dir}/results/{model_name}/results_{timestamp}.json`. [Here is an
 example of a result file](#example-of-a-result-file). The output path can be
 any [fsspec](https://filesystem-spec.readthedocs.io/en/latest/index.html)
 compliant path (local, s3, hf hub, gdrive, ftp, etc).
 
-To save the details of the evaluation, you can use the `--save_details`
-argument. The details will be saved in a parquet file
+To save the details of the evaluation, you can use the `--save-details`
+option. The details will be saved in a parquet file
 `{output_dir}/details/{model_name}/{timestamp}/details_{task}_{timestamp}.parquet`.
```

Comment:
Side note: in the future I believe it would be good to have something more homogeneous

Reply:
agreed !